lib/go/test/test.go

```diff
@@ -244,7 +244,9 @@ func metadataDict(dict map[string]string) cadence.Dictionary {
 	pairs := []cadence.KeyValuePair{}
 
 	for key, value := range dict {
-		pairs = append(pairs, cadence.KeyValuePair{Key: cadence.NewString(key), Value: cadence.NewString(value)})
+		k, _ := cadence.NewString(key)
```

Comment:
These changes aren't related to the secure Cadence work but rather an older version of the SDK that we hadn't updated in a while